entrypoint.sh

```diff
@@ -50,11 +50,8 @@ tmp_dir=$(mktemp -d -t ci-XXXXXXXXXX)
     git pull "$GIT_REPOSITORY_URL"
 )
 
-debug "Enumerating contents of $1"
-for file in $(find $1 -maxdepth 1 -type f -name '*.md' -execdir basename '{}' ';'); do
-    debug "Copying $file"
-    cp "$1/$file" "$tmp_dir"
-done
+debug "Syncing contents of $1 to repository"
+rsync -avzr --delete --exclude='.git/' "$1" "$tmp_dir"
```

Comment:
Not sure what the original intent was here, but I found that `"$1/"` copies the contents of the directory over. Without the forward slash, it seems like `rsync` would copy over the directory itself.

Reply:
I mean, you can see the original intent from the lines of code being removed. As far as I can tell, this `rsync` command does what I want, which is to synchronize the contents of the directory into the repo checked out into a temporary directory.